buildingsync_asset_extractor/config/asset_definitions.json

```diff
@@ -112,6 +112,30 @@
       "export_name": "Hot Water System Fuel Type",
       "type": "custom",
       "export_units": false
+    },
+    {
+      "parent_path": "/BuildingSync/Facilities/Facility/Systems/HVACSystems/HVACSystem/HeatingAndCoolingSystems/HeatingSources/HeatingSource",
+      "key": "PrimaryFuel",
+      "name": "ElectrificationPotential",
+      "export_name": "Heating Electrification Potential",
+      "type": "custom",
+      "export_units": true
+    },
+    {
+      "parent_path": "/BuildingSync/Facilities/Facility/Systems/DomesticHotWaterSystems/",
+      "key": "PrimaryFuel",
+      "name": "ElectrificationPotential",
+      "export_name": "Domestic HotWater System Electrification Potential",
+      "type": "custom",
+      "export_units": true
+    },
+    {
+      "parent_path": "/BuildingSync/Facilities/Facility/Systems/HVACSystems/HVACSystem/HeatingAndCoolingSystems/CoolingSources/CoolingSource",
+      "key": "PrimaryFuel",
+      "name": "ElectrificationPotential",
+      "export_name": "Cooling Electrification Potential",
+      "type": "custom",
+      "export_units": true
     }
   ]
```

Comment:
Do you want to add other systems in another PR or together in this one? I suggest add CookingSystem in here as well because it has all the same elements required.